hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/lib/package-info.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
The license header should be just a plain comment instead of JavaDoc.

Reply:
Thank you very much for your help in reviewing the code!  I re-added the comment. Can you help to review this PR again?